util/env_test.go

```
@@ -35,6 +41,29 @@ func TestGetEnvDefault(t *testing.T) {
 	assert.Equal(t, "fallback", GetEnvDefault("TEST_DEFAULT_MISSING", "fallback"))
 }
 
+func TestGetSecretEnvDefaultDoesNotPrintValueInDebugMode(t *testing.T) {
+	t.Setenv("NEXTTRACE_DEBUG", "1")
+	t.Setenv("TEST_SECRET_KEY", " secret-token ")
+
+	oldStdout := os.Stdout
+	r, w, err := os.Pipe()
+	require.NoError(t, err)
+	defer r.Close()
```

golangci-lint (2.12.2)

[error] Error return value of `r.Close` is not checked

(errcheck)